docker-compose.yml

```diff
@@ -12,11 +12,8 @@ services:
       - POSTGRES_PASSWORD=trees3r4t
       - POSTGRES_DB=postgres
       - DB_NAME=treedb
-      # - DB_FILE=treedb.sql
       - DB_USER=trees
       - DB_PASSWORD=trees3r4t
```

Comment:
Can we discuss generating passwords for developers?
Obviously this issue was not introduced in this pull.

Reply:
Agreed. Slack first I think